book/website/reproducible-research/vcs/vcs-git-interactive.md

```diff
@@ -0,0 +1,19 @@
+(rr-vcs-interactive)=
+# Interactive, Visual Git
+
+One of the biggest challenges to getting started with Git can be understanding the file and directory structure and complexity of branches, commits, remotes, etc. and how they interact, particularly on the command line with limited visual representations.
```

Comment:
Hi! We'll need to substitute the "etc." here with something else to pass our check for Latin phrases. These are useful, but are sometimes more difficult to comprehend than more usual alternatives. There's a bit more info [here](https://the-turing-way.netlify.app/community-handbook/style.html?highlight=style#avoid-latin-abbreviation).

Reply:
Thanks for the reminder @inwaves!